src/views/patch.rs

```diff
@@ -85,11 +85,15 @@ pub fn get_change_graph(
         });
         for bg_edge in bg_edges {
             let edge = *edges_by_id.get(&bg_edge.edge_id).unwrap();
-            graph.add_edge(
-                edge.source_node_id,
-                edge.target_node_id,
-                (edge.source_coordinate, edge.target_coordinate),
-            );
+            if let Some(weights) = graph.edge_weight_mut(edge.source_node_id, edge.target_node_id) {
```

Comment:
the existing setup only allowed 1 set of source/target coords from a given node src -> target. this makes it a vec to store all connections.